method-proposals/PROPOSAL-did-cheqd.md

```diff
@@ -0,0 +1,142 @@
+| **Scalability and performance** |
+| Support for between 5000 and 10,000 transactions per second, with multiple transactions allowed and batched per block |
+| **Ease of implementation and use** |
+| Muiltiple implementation routes, via SDKs and APIs as well as full support in the Universal Registrar and Resolver. Clear documentation and tutorials written for each seperate integration pathway  |
+| **Community adoption and support** |
+| Over 300,000 DIDs on testnet and support in four of the largest industry SDKs, as well as over ten identity applications |
+| **Compliance with relevant regulations and best practices** |
+| Yes, and actively working with EU on establishing best practices for using DLT as trust infrastructure for the EU Digital Identity market. |
+| **Global government-approved crypto** |
+| The DID Method itself uses the latest approved NIST and FIPS cryptography (ECDSA and EdDSA) used by many Western nation-states with ability to support cryptography backed by other Eastern nation-state governments. The DID Method is defined to support evolving the cryptography used by the DID Method through the versioning of the specification and DIDs using the specification.  |
+| **Privacy-preserving crypto** |
+| Yes, supports BBS privacy-preserving cryptographic key formats embedded within the DIDDoc. Any DID-Core compatible key formats can be defined in verification method relationships within the DID Documents, even if they are not natively validated against by the underlying cheqd network. |
+| **Digitally signed cryptographic log of changes to the DID Document** |
+| Yes. |
+| **Multi-factor binding to DNS** |
+| No. |
+| **Specification with multiple implementers** |
+| Yes. |
+| **Scope/domain of the types of entities/subjects addressed/named by a particular method** |
+| Expected to be used primarily for organizational entities, but that is not limited by the specification. |
+| **Estimate of the daily transaction volume of each scope/domain** |
+| Yes, daily metrics are actively recorded and provided via services such as [GoDiddy Stats](https://stats.godiddy.com/cheqd). |
+| **DID Methods that do not serve the needs of a particular company or government** |
+| The cheqd Network and associated DID Method is separate from cheqd as a profit making entity and is controlled by its decentralized governance framework. As such, it is a public resource which may be used via open source libraries without a tie to any company or government. |
+| **Governance: Clear frameworks for updates, dispute resolution, and decision-making** |
+| Established decentralized governance framework, with clear documentation, discussion forum and simple voting process. There have been 60 proposals on mainnet as of January 2025 |
+| **Usability: Simple implementation for developers** |
+| Yes, as evidenced by developers building across SDKs, APIs and Registrar implementations |
+| **Sustainability: Energy efficiency and eco-friendly infrastructure** |
```

Comment:
Specifically worth calling out now PoW and minimum hardware requirements/corresponding power estimates instead; unsure about Regen

Reply:
@ankurdotb could you please edit this one yourself as you're more aware of these than I am